fanficfare/defaults.ini

```diff
@@ -2323,6 +2323,20 @@ add_to_comma_entries:,pages,readers,reads,favorites,comments
 
 website_encodings: utf8:ignore, Windows-1252, iso-8859-1
 
+[readonlymind.com]
```

Comment:
Should also be added to `calibre-plugin/plugin-defaults.ini`.

Reply:
Ah, didn't see that ini, as I don't use calibre. Sure.